python/az/aro/azext_aro/custom.py

```diff
@@ -129,10 +130,21 @@ def aro_list_credentials(client, resource_group_name, resource_name):
 
 def aro_update(client, resource_group_name, resource_name, worker_count=None,
                no_wait=False):
+    current = client.get(resource_group_name, resource_name)
+
+    if len(current.worker_profiles) != 1:
+        raise CLIError(
+            "Can only perform update on a cluster with 1 worker profile. This cluster has %d"
```

Comment:
I'm working under the belief that MSFT mandate that error messages must end with a `.`.
If feasible, I think it's worth aiming for single sentence error messages - e.g. "Cannot update cluster with %d worker profiles." ?